readthedocs_theme/templates/includes/topnav.html

```diff
@@ -95,6 +111,11 @@
 
               <div class="divider"></div>
 
+              <div data-analytics="login-modal" class="header">Sign Up</div>
+              {{ menu_sign_up() }}
+
+              <div class="divider"></div>
```

Comment:
We should still open the modal, even if it is on mobile/tablet view.

The modal was designed to work on all view ports:

![image](https://github.com/readthedocs/website/assets/1140183/068eaacf-710b-47e9-a069-75812a54a306)